### ISSUE TITLE
Unable to install JB in WSL2

### ISSUE THREAD

**Issue by albittar** [jsonnet-bundler/jsonnet-bundler#178]
I am following the steps in this repo to install Jsonnet Bundler. However, I think the installation process is not going through. I am using WSL2. Nothing happens or printed out after running the following command: 

go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest

uname -a output:
Linux CBR-53KT0T3 5.10.102.1-microsoft-standard-WSL2 #1 x86_64 x86_64 x86_64 GNU/Linux

**Comment by PatTheSilent** [comment]
I don't believe this is a problem with WSL2. I've been running WSL2 for months and have a functional install of `jb` 


```
Linux DESKTOP-2FA2PSQ 5.15.133.1-microsoft-standard-WSL2 #1 SMP Thu Oct 5 21:02:42 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```
